### DISCUSSION TITLE
Journals that accept executable content?

### DISCUSSION THREAD

**Discussion by mbercx** [executablebooks/meta#1167] [Q&A]
I've been working with some colleagues on a Jupyter book comparing various materials science workflow engines (mostly using DFT) and their implementation of a simple equation of state workflow. It's pretty clear that for this kind of content, publishing a Jupyter book would be the natural choice, as we want readers to be able to execute the examples for the various workflow engines.

However, it seems hard to find journals that are open to this. Is anyone aware of academic journals that accept executable content?

**Comment by agoose77** [comment]
@agahkarakuzu I think neurolibre qualifies for this?

@rowanc1 any thoughts on this w.r.t Curvenote?

**Reply by agahkarakuzu** [reply]
@agoose77 indeed! Even though we register the content as preprint, that's what neurolibre publishes.

**Reply by mbercx** [reply]
Very interesting, thanks! Seems perfect for publishing our preprint indeed. I'll have a closer look and discuss with my collaborators.

**Reply by agahkarakuzu** [reply]
@mbercx great! Feel free to reach out info@neurolibre.org, happy to discuss how to proceed.

**Comment by rowanc1** [comment]
@mbercx if you are interested in setting up a journal for computational articles/books that uses JupyterBook/MySTMD, that is what Curvenote does:
https://curvenote.com
We are running Scipy Proceedings this year off of MyST:
https://proceedings.scipy.org
In addition to the computational reading experience, Curvenote supports communities with the archiving and standards that are needed to integrate with the scholarly publishing ecosystem. Happy to chat more if you want @mbercx!

**Reply by mbercx** [reply]
Looks great! I'm not sure if I'm ready to start a journal ^^, but will explore already existing options a bit and reach out via email.
